src/components/gui/gui.jsx

```diff
@@ -87,25 +95,31 @@ const GUIComponent = props => {
                         </Tabs>
                     </Box>
 
-                    <Box className={styles.stageAndTargetWrapper} >
-                        <Box className={styles.stageMenuWrapper} >
-                            <GreenFlag vm={vm} />
-                            <StopAll vm={vm} />
-                        </Box>
+                    <MediaQuery minWidth={layout.fullSizeMinWidth}>
+                        {isFullSize => (
```

Comment:
From offline: This is surprising, let's move it to the line above so it's clear it's related to `MediaQuery`.